Source/WebCore/html/HTMLFrameOwnerElement.h

```diff
@@ -66,14 +66,15 @@ class HTMLFrameOwnerElement : public HTMLElement {
     virtual bool shouldLoadFrameLazily() { return false; }
     virtual bool isLazyLoadObserverActive() const { return false; }
 
+    bool isKeyboardFocusable(KeyboardEvent*) const override;
```

Comment:
This could be under `protected:`

Reply:
Sure, will move. I am expecting this to fail on 'iOS-wk2' due to keyboard event, so I will add 'Platform Specific expectation' and this part in one go. :-)

Reply:
Done.